### ISSUE TITLE
[xenial] hotspot not working for Meizu Pro5

### ISSUE THREAD

**Issue by mattbel10** [ubports/ubuntu-touch#806]
### Description of the feature
For the Meizu Pro5, the hotspot feature is simply not working either if I set the password or not.

### Illustrations

- Device: Meizu Pro5 64 GB
- Channel: 16.04 devel
- Build: 2018-07-30

### Steps to reproduce

- Be sure to have a cellular data available and running from your sim card (I got a 4G connection enabled). 

- Go to 'System settings' and press the 'change password' button in order to set for the first time a hotspot connection to be shared among your phone and other devices. Flagging the password request or not will not change the result.

- Once modified the hotspot credentials and went back to the main hotspot window, enable the hotspot by moving the available toggle to the right (it will show a green color for 'enabled'). A proper icon for the hotspot-indicator will be displayed on the upper edge of the phone.

### Expected behavior

All the machines with an embedded wi-fi card should detect the new hotspot access point and should be able to connect to it.

### Actual behavior

None of the devices at my disposal can detect the new hotspot access point (I tried with an Android phone, a BQ E5 HD 15.04 UBports edition and the Dell Inspiron laptop running Ubuntu 16.04 LTS).

### Logfiles and additional information

I don't see any specific logfile related to the hotspot feature in the Logviewer app by neothethird but if needed other sort of logs please point those to me.

**Comment by TartanSpartan** [comment]
@Flohack74 the device should be turbo, not arale.

@mattbel10 on vivid for turbo, I can only trigger the hotspot by turning it on from the indicator, going to system settings, Hotspot, Change Hotspot Setup, change to "Require a password", hit Change, Change Hotspot Setup, turn "Require a password" off again, hit Change again and then finally when I turn WiFi on for client devices, I can see the hotspot (but of course this is not password protected and leads to the threat of unauthorised users hijacking my mobile internet data). Does this workaround work for you on Xenial?

**Comment by mattbel10** [comment]
@TartanSpartan I tried your way also but even if the hotspot indicator is correctly shown when enabled on the top edge of the phone, it seems there is no actual signal sent over-the-air because my laptop is always unable to detect the device among all the other wi-fi access points.

**Comment by mattbel10** [comment]
I noticed an other side effect on the OS after Anbox installation: the sim card isn't recognized at all so I cannot use my device as an actual phone anymore.

**Comment by wesertrainer** [comment]
I have to report an unstable operation of the hotspot connection, too. Connection can be established only for some minutes, sometimes less than an minute.

 Device turbo, 16.04/rc, Version 14

**Comment by tomoqv** [comment]
I can confirm that the Pro 5 doesn't display any wifi network on RC 2018-W48. I have tried to find the network from two different devices, but nothing. The indicator icon looks as if it is not transmitting.

**Comment by tomoqv** [comment]
I can add that wifi did not come back after turning hot spot off. I had to reboot to get the Pro 5 to find wifi networks again and connect to my known network. Maybe a separate issue, just thought I'd mention it.

**Comment by hummlbach** [comment]
mhmm, I cannot confirm any of these. Hotspot is stable with my turbo (using it in the train regularily to have internet on my laptop...). If i turn it off at home it switches from mobile data connection over to my wifi network and also displays many other wifis available just fine.

**Comment by mattbel10** [comment]
My Pro5 may likely have issues because it was the chinese version manually converted, after a tricky procedure, to International. Would you kindly confirm to me guys if this is the case for you also? And if yours is the 64GB version?

**Comment by hummlbach** [comment]
@tomoqv yes the an box kernel most likely fails regarding hotspot and would really like to know why but have no clue yet. Even the kernel found here https://github.com/meizuosc/m576/tree/ubuntu is compiled as is and is flashed according to the instructions, the hotspot doesn't work anymore afterwards... :-(
@mattbel10 how to see whether I have the chinese version? At the back of my phone there are some chinese characters at least... (after pro 5 td-lte)

**Comment by tomoqv** [comment]
I did a flash & wipe of my Pro5 before X-mas and hotspot seems to work fine, although the hotspot indicator icon looks as if there is very poor or no signal strength.

**Comment by mattbel10** [comment]
@hummlbach interesting that link to the kernel. Do you think it is any worth a try to build and flash that kernel onto my Pro5 to see what happens, even if I don't have Anbox installed? I don't know how you could identify if your phone is chinese or not...I guess it strongly depends on how you got it in your hand, I mean mine was delivered from Hong Kong with chinese Flyme installed.

**Comment by hummlbach** [comment]
I think an update ota reflashes the (actual working) kernel... if you did that recently, then the problem is most likely somewhere else...

**Comment by cibersheep** [comment]
I can confirm on vegetahd RC 2019-W25

**Comment by Amooti73** [comment]
I have been unable to use my Nexus 5 with OTA-15 as a hotspot.  I have also tried both with and without a password. The hotspot symbol shows up in the bar at the top and I have cellular data enabled, but not success so far.
